packages/brand-shape/src/Blob.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { getByTestId, render } from '@testing-library/react';
+
+import { Blob } from '.';
+
+describe('packages/blob', () => {
+  test('Blob renders', () => {
+    const { container } = render(
+      <Blob
+        data-testid="blob"
+        shape={[
+          ['o', ' ', ' ', ' '],
+          [' ', 'o', ' ', ' '],
+          [' ', 'o', 'o', ' '],
+          ['o', ' ', ' ', 'o'],
+        ]}
+      />,
+    );
+
+    const blob = getByTestId(container, 'blob');
+    expect(blob).toBeInTheDocument();
+  });
+
+  // TODO: add more tests
```

Comment:
probably would want to lean on Chromatic more than functional tests - just wouldn't make sense for a package like this